backend/src/index.ts

```diff
@@ -384,6 +385,17 @@ handlers[API_PROFILE] = async (input) => {
   };
 };
 
+handlers[API_PROFILE_CREATE] = async (input) => {
```

Comment:
I guess this should be API_PROFILE_UPDATE

Reply:
I was actually going to suggest this!`